README.md

```diff
@@ -87,3 +87,45 @@ React-Native-Zapp-Bridge is a util package that enable you to:
       .catch(error);
   }
   ```
+
+### Zapp login
+1. import:
+```javascript
+import { NativeModules } from 'react-native';
+```
+2. check if user is logged in
+```javascript
+NativeModules.ZappLogin.isUserLoggedIn();
+```
+Returns:
+Promise - resolves:
+`{ isUserLoggedIn: bool }`
+
+3. login user
+```javascript
+NativeModules.ZappLogin.login(params);
+```
+parameters:
+`params: Object`
```

Comment:
What are the available parameters ?

Reply:
Yeah I get it. It's just unnatural - when you have a login method, you expect to be able to send credentials and get a token back. But the problem is not in the documentation, it's in our architecture of the login feature. And that's a problem for another day unfortunately